PolicyServerElasticsearchContainer/pom.xml

```diff
@@ -418,49 +410,26 @@
                             <alias>policy-server</alias>
                             <name>${container.policyserver.name}</name>
                             <build>
-                                <from>${container.policyelasticsearchbase.name}</from>
+                                <from>${container.policyelasticsearchtomcatbase.name}</from>
                                 <labels>
                                     <Build.Number>${project.version}</Build.Number>
                                     <Build.Date>${maven.build.timestamp}</Build.Date>
                                     <Git.Branch>${git.branch}</Git.Branch>
                                     <Git.Commit>${git.revision}</Git.Commit>
                                 </labels>
-                                <!-- Expose port for tomcat -->
-                                <ports>
-                                    <port>8080</port>
-                                </ports>
                                 <env>
-                                    <CATALINA_HOME>/opt/tomcat</CATALINA_HOME>
+                                    <CATALINA_HOME>/usr/share/tomcat</CATALINA_HOME>
                                     <PATH>$PATH:$CATALINA_HOME/bin</PATH>
                                 </env>
-                                <!-- Run application through tini -->
-                                <entryPoint>
-                                    <exec>
-                                        <args>/opt/tini-${tini.version}.exe</args>
-                                        <args>--</args>
-                                    </exec>
-                                </entryPoint>
                                 <runCmds>
-                                    <runCmd>mv /opt/${fileSet.outputDirectory}/apache-tomcat-* /opt/tomcat</runCmd>
-                                    <runCmd>mkdir /var/corepolicy</runCmd>
-                                    <runCmd>mkdir ${engine.environmentcache.location}</runCmd>
```

Comment:
What are these two `mkdir` lines that are being removed?  Are they no longer required?

Reply:
I do not know what these mkdir lines are for, the directories created don't appear to be used anywhere.

Reply:
Let's not risk it